resolwe/api_urls.py

```diff
@@ -23,6 +24,9 @@
 api_router.register(r"descriptorschema", DescriptorSchemaViewSet)
 api_router.register(r"storage", StorageViewSet)
 api_router.register(r"upload_config", UploadConfig, basename="upload_config")
+api_router.register(
+    r"background_task", BackgroundTaksViewSet, basename="backgroundtask"
```

Comment:
Rename this to `task` to simplify the urls.

Reply:
Done.